sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -3,11 +3,13 @@
 ## 7.7.0-beta.1 (Unreleased)
 
 ### Features Added
+- Added support for sending/receiving messages with `Duration`, `OffsetDateTime` and `URI` in `applicationProperties`.
 
 ### Breaking Changes
 
 ### Bugs Fixed
 - Removed the incorrect use of lock primitives from `ServiceBusMessageBatch.tryAddMessage()` implementation and documented that this API is not thread-safe. ([#25910](https://github.com/Azure/azure-sdk-for-java/issues/25910))
+- Fixed a bug that messages come from .net SDK with `DescribedType` cannot be deserialized. ([#26065](https://github.com/Azure/azure-sdk-for-java/issues/26065))
```

Comment:
I think ".net SDK" is not official name.

Reply:
azure-sdk-for-net could be better?